api/src/main/java/com/datastrato/gravitino/authorization/AuthorizationRole.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2024 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+package com.datastrato.gravitino.authorization;
+
+/** Authorization Role interface. */
+public interface AuthorizationRole {
+  /**
+   * Create a role.
```

Comment:
Can you please add more comments about when these interfaces like `createRole` and others will be called.

Reply:
DONE.